src/main/java/org/elasticsearch/action/admin/cluster/state/ClusterStateRequest.java

```diff
@@ -137,5 +144,6 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeBoolean(metaData);
         out.writeBoolean(blocks);
         out.writeStringArray(indices);
+        indicesOptions.writeIndicesOptions(out);
```

Comment:
we need to add a version check here on the 1.x branch, to make sure that we maintain bw compatibility:
```
if (out.getVersion().onOrAfter(Version.1_5_0)) {
    indicesOptions = IndicesOptions.writeIndicesOptions(out);
}
```